### PR TITLE
Release 12.0.2

### PR DESCRIPTION
**Pull request by botanical**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Review by npauzenga — APPROVED** [review]
LGTM

File: website/versions.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+  "v12.0.2",
```
Comment (npauzenga) [inline website/versions.json:2]: I don't think we want the website version updated here... 🤔. Double-check me but I think we only do this for forward versions:

https://github.com/nasa/cumulus/blob/master/docs/development/release.md#8-cut-new-version-of-cumulus-documentation

Comment (botanical) [reply website/versions.json]: https://github.com/nasa/cumulus/pull/3076/commits/cf019ba3f573988756c9eb373df7090bf6480c68

Comment (npauzenga) [reply website/versions.json]: Thanks!

**Pull request merged**